vip-api/src/main/java/fr/insalyon/creatis/vip/api/business/ApiBusiness.java

```diff
@@ -28,22 +29,23 @@ public AuthenticationInfo authenticate(AuthenticationCredentials authCreds)
             throws ApiException {
         String username = authCreds.getUsername(), password = authCreds.getPassword();
         logger.debug("Verifying credential for " + username);
-        signin(username, password);
+        User user = signin(username, password);
```

Comment:
We need to bring back the previous behavior as default and to add the one we need here.